sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -468,6 +468,9 @@ RUN apt-get install -y xsltproc
 # Install dependencies for isc-dhcp-relay build
 RUN apt-get -y build-dep isc-dhcp
 
+# Install dependencies for isc-dhcp-server build
+RUN apt-get -y build-dep isc-dhcp-server
```

Comment:
This isn't needed, since `isc-dhcp` is the source package name, and that's specified above.